onnxruntime/core/optimizer/conv_activation_fusion.cc

```diff
@@ -56,26 +58,24 @@ bool ConvFusionDataTypeCheck(const Node& conv_node) {
     }
   }
   if (node_ep == kCpuExecutionProvider) {
-#ifdef MLAS_F16VEC_INTRINSICS_SUPPORTED
     if (!HasElementDataType(*conv_node.InputDefs()[0], ONNX_NAMESPACE::TensorProto_DataType_FLOAT) &&
-        !HasElementDataType(*conv_node.InputDefs()[0], ONNX_NAMESPACE::TensorProto_DataType_FLOAT16)) {
+        !HasElementDataType(*conv_node.InputDefs()[0], ONNX_NAMESPACE::TensorProto_DataType_FLOAT16) &&
+        support_fp16) {
```

Comment:
now, if `support_fp16 == false`, this won't return false. I guess `if (support_fp16)` should replace the `#ifdef MLAS_F16VEC_INTRINSICS_SUPPORTED` from before right?

Reply:
yes